packages/toggle/src/component.tsx

```diff
@@ -38,18 +43,44 @@ export function Toggle(props: ToggleProps) {
 
   const helpId = props.helpText ? `${id}__hint` : undefined;
   const isInvalid = props.invalid;
-  const isRadioGroup = props.type === 'radio' || props.type === 'radio-button';
-
+  const isRadio = props.type === 'radio';
+  const isCheckbox = props.type === 'checkbox';
+  const isRadioButton = props.type === 'radio-button';
+  const isRadioGroup = isRadio || isRadioButton;
   const isControlled =
-    props.selected !== undefined || props.checked !== undefined;
+    props.selected !== undefined || props.checked !== undefined;  
+
+  const labelClasses = classNames({
+    [ccToggle.label]: !isRadioButton,
+    [ccToggle.focusable]: !isRadioButton,
+    [ccToggle.noContent]: !props.indeterminate,
+    [ccToggle.indeterminate]: props.indeterminate,
+    [`${ccToggle.radio} ${ccToggle.labelRadioColors}`]: isRadio,
+    [ccToggle.radioInvalid]: isRadio && isInvalid,
+    [`${ccToggle.checkbox} ${ccToggle.labelCheckboxColors}  ${ccToggle.checkboxChecked}`]: isCheckbox,
+    [ccToggle.icon]: isCheckbox && !props.indeterminate,
+    [ccToggle.checkboxInvalid]: isCheckbox && isInvalid,
+    [ccToggle.scLabel]: isRadioButton,
+    [ccToggle.scLabelJustified]: props.equalWidth,
```

Comment:
I think these classes should be changed in component-classes to include `flex-grow-1` for labels to be of equal width.